docs/diagnostics/PX1078.md

```diff
@@ -0,0 +1,50 @@
+# PX1078
+
+This document describes the PX1078 diagnostic.
```

Comment:
@EAndrosova please check the documentation for the new diagnostic

Reply:
@acasj I've got this file with fixes of the documentation from @EAndrosova.
Please replace the documentation with it.
[PX1078.md](https://github.com/user-attachments/files/18198208/PX1078.md)